scripts/create_empty_scalars.py

```diff
@@ -82,41 +101,57 @@ def expand_scalars(df, column, where, expand):
     return _df_wo_cc
 
 
+def add_wacc(sc):
+    sc = sc.append(
+        {
+            "scenario_key": scenario_specs["name"],
+            "var_name": "wacc",
+            "carrier": "ALL",
+            "region": "ALL",
+            "tech": "ALL",
+            "type": "ALL",
+            "var_unit": "None",
+        },
+        ignore_index=True,
+    )
+
+    return sc
+
+
+def save_empty_scalars(sc, path):
```

Comment:
This function appends to an existing file. That can make it difficult to keep track of things. Do we really want to do this? If yes, it needs to be communicated more prominently (name of the function, docstrings). But first, think if we really need this!

Reply:
Yes, this only makes sense if I get the loop implemented over the scenarios. Then there would be a single `empty_scalars.csv` which is updated with every iteration over the scenario. I have therefore left it in.